app/api/admin_statistics_api/users.py

```diff
@@ -47,23 +47,23 @@ def verified_count(self, obj):
     def unverified_count(self, obj):
         return get_count(User.query.filter_by(is_verified=False, is_super_admin=False, is_admin=False))
 
-    def get_all_user_roles(self, role_name):
+    def get_all_user_role(self, role_name):
         role = Role.query.filter_by(name=role_name).first()
-        uers = UsersEventsRoles.query.join(UsersEventsRoles.event).join(UsersEventsRoles.role).filter(
-            Event.deleted_at.is_(None), UsersEventsRoles.role == role)
-        return uers
+        newquery = User.query.join(UsersEventsRoles.user).join(UsersEventsRoles.role).filter(
+            UsersEventsRoles.role==role).distinct()
+        return newquery
 
     def organizer_count(self, obj):
-        return get_count(self.get_all_user_roles('organizer'))
+        return self.get_all_user_role('organizer').count()
```

Comment:
Use get_count

Reply:
@iamareebjamal `get_count` function is returning a wrong count (of 8 TBP) instead of desired 3

Reply:
If you pass `self.get_all_user_role('organizer')`?

Reply:
@iamareebjamal Yes, it's giving 8 then